### DISCUSSION TITLE
blog/v1-30-is-here/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#291] [Page comments]
# blog/v1-30-is-here/

Happy New Year! It’s been over two months since the last version, but we’ve been working hard on several exciting new features.

https://scratchaddons.com/blog/v1-30-is-here/

**Comment by CST1229** [comment]
THE BLOG IS BACK!

**Reply by progersbolanos-cmyk** [reply]
wow

**Reply by progersbolanos-cmyk** [reply]
im gay

**Reply by progersbolanos-cmyk** [reply]
t
